### ISSUE TITLE
Jupyter-like focus indicator missing when selecting notebook layout

### ISSUE THREAD

**Issue by kieferrm** [microsoft/vscode#125339]
![k1HRihwnFe](https://user-images.githubusercontent.com/4674940/120551000-69187800-c3aa-11eb-8aec-660e39295959.gif)

**Comment by roblourens** [comment]
Do you want to see an "inactive" (gray color) focus indicator or do you want to see the blue indicator all the time even when the editor is not focused? Do you want to see it when the editor is focused? Trying to figure out how the behavior compares to the border indicator. I can't decide what I want.

**Comment by rebornix** [comment]
> Do you want to see it when the editor is focused

this is tracked in https://github.com/microsoft/vscode/issues/125338

> Do you want to see an "inactive" (gray color) focus indicator or do you want to see the blue indicator all the time even when the editor is not focused

Right now we didn't show the inactive focus indicator at all when the editor is blurred, which should be fixed.

Verifier, please test with following settings

```
"notebook.cellFocusIndicator": "gutter",
"workbench.colorCustomizations": {
        "notebook.inactiveFocusedCellBorder": "#ff0000"
    },
```

click on a cell, focus outside of the notebook editor, and make sure you see the red cell border.
